index.js

```diff
@@ -70,8 +70,14 @@ function request (params, fn) {
 
   // POST API request body
   if (params.body) {
-    req.send(params.body);
-    debug('API send POST body: ', params.body);
+    // GET requests don't support body; convert to querystring
```

Comment:
[Technically according to HTTP/1.1](http://stackoverflow.com/q/978061), GET requests with a body are allowed, just extremely rare.